source/introduction.rst

```diff
@@ -0,0 +1,13 @@
+###################
+Introduction
+###################
+
+.. toctree::
+   :maxdepth: 2
+
+   what-is-ethereum.rst
+   the-homestead-release.rst
+   solidity-in-depth.rst
```

Comment:
I don't see why not. the toctree is a common thing for the the larger sub-pages of a guide. It won't be the *only* thing necessarily, but I was putting it there at first so there wouldn't be a blank page.